README.md

```diff
@@ -63,6 +63,20 @@ coming soon
 
 # Working on Docs
 
+## Datadog Staff
+
+* Always branch off of master
+* Create a branch `slack_handle/your_feature` if you would like to create a preview site and run tests
+* Never commit directly to master
```

Comment:
This seems redundant with the first item. Maybe combine them both: "Always branch off of master; never commit directly to master"